docs/Users_Guide/getting_started.rst

```diff
@@ -0,0 +1,356 @@
+***************
+Getting Started
+***************
+
+This chapter reviews some important things to consider before starting
+a METplus project.
+
+Questions to Consider
+=====================
+
+Questions to ask when applying METplus to a new testing and evaluation project
+------------------------------------------------------------------------------
+
+If a user is new to developing a verification or evaluation system or
+to using METplus, there are questions that should be considered to help
+determine which tools to use and how to set up METplus.
+
+* First and foremost, what are the questions need to be answered
+  with this testing and evaluation project?
+  
+* What type of forecasts and type of observations will be used and how
+  they can/should be matched?
+  
+  * What attributes of the forecast should be evaluated?
+     
+  * What is the standard for comparison that provides a reference level
+    of skill (e.g., persistence, climatology, reference model)?
+     
+  * What is the geographic location of the model data being evaluated?
+    Are there specific areas of interest for the evaluation?
+     
+  * What domain should be used to evaluate on the model domain, 
```

Comment:
I think this one is supposed to read like "What domain should be used to evaluate on: the model domain, observation domain (if gridded), or some other domain?" The model and observations need to be evaluated on the same domain, but users can determine what domain that is. How it's currently written sounds like each domain is evaluated separately.

Reply:
I used your wording and also changed the matching question lower down.